### DISCUSSION TITLE
Dedicated Discord server?

### DISCUSSION THREAD

**Discussion by danielhenrymantilla** [danielhenrymantilla/rust-uninit#34] [Ideas]
I know we have Github Discussions already, but let's be honest: it's not a common chat platform. Even though @kupiakos and I are currently the main maintainers of this crate, there is decent interest from the `zerocopy` effort in this crate as well, so having a space where they could join and we could discuss about `uninit` design or ideäs could be neat. Thoughts?

**Comment by kupiakos** [comment]
I like it for quick comms, though it's harder to search and less integrated
